### PR TITLE
Change XZC to FIRO

### PR DESCRIPTION
**Pull request by peachbits**

### CODE REVIEW

File: src/info/zcoin.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const bcoinInfo: BcoinCurrencyInfo = {
 
 const engineInfo: EngineCurrencyInfo = {
   network: 'zcoin',
```
Comment (paullinator) [inline src/info/zcoin.js:28]: I believe the network name needs to change to "firo". Confirm with @thehobbit85, but I believe this affects the address URI format

**Review by paullinator — CHANGES_REQUESTED** [review]

**Review by paullinator — CHANGES_REQUESTED** [review]
Also, symbolImage* should change to a different file name so that it does not change the image for people who have not updated the app.

File: src/info/zcoin.js (hunk 1 above)
Comment (peachbits) [reply src/info/zcoin.js:29]: I think the simpler solution is to use uriPrefix. Changing the network name requires additional changes to work that I haven't investigating besides seeing that the wallets don't load with network = firo. Using uriPrefix comes with the side benefit of being able to scan both zcoin: and firo: uris

Comment (paullinator) [reply src/info/zcoin.js:29]: I was not aware of URI prefix. If that is available then yes definitely use it

**Review by paullinator — APPROVED** [review]

**Pull request merged**